model/lists/ListServiceProvider.php

```diff
@@ -26,18 +26,27 @@
 use oat\tao\model\Lists\Business\Specification\RemoteListClassSpecification;
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
+use function Symfony\Component\DependencyInjection\Loader\Configurator\env;
 use function Symfony\Component\DependencyInjection\Loader\Configurator\service;
 
 class ListServiceProvider implements ContainerServiceProviderInterface
 {
     public function __invoke(ContainerConfigurator $configurator): void
     {
         $services = $configurator->services();
+        $parameters = $configurator->parameters();
+
+        $parameters->set('MAX_ALLOWED_LIST_ELEMENTS_LIMIT', 1000);
```

Comment:
MAX and LIMIT as the same. Allowed is also implicit since you are defining a config for this property, no?

```suggestion
        $parameters->set('LIST_ELEMENTS_LIMIT', 1000);
```

Reply:
Closing, cause it is not mandatory